src/components/select/SelectReps/index.tsx

```diff
@@ -73,42 +73,67 @@ const SelectReps: React.FC<SelectRepsProps> = (props) => {
       <FormLabel sx={{ display: "flex", alignItems: "center", gap: 1 }}>
         Reps: {props.reps}
         {props.isAmrap && (
-          <Typography variant="body2" color="primary">
+          <Typography variant="body2" color="secondary">
             (AMRAP)
           </Typography>
         )}
       </FormLabel>
-      <ToggleButtonGroup
-        color="primary"
-        value={props.reps}
-        exclusive
-        onChange={api.handleToggleChange}
-        size="small"
-        aria-label="Select Reps"
-        sx={{ mb: 1 }}
```

Comment:
I actually did change this, but the margin bottom shouldn't have been set anyway, AI does shit like this constantly. It just decides to add random custom formatting that doesn't serve _ANY_ purpose.